### ISSUE TITLE
additional "axis" parameter and different default values of keras.layers.LayerNormalization

### ISSUE THREAD

**Issue by pbecker93** [tensorflow/tensorflow#33577]
Hello,

**System Info**

Tensorflow Version 2.0 

**Describe the feature and the current behavior/state.**

The current (tensorflow 2.0) implementation of keras.layers.LayerNormalization accepts one argument, "axis", specifying the axis that should be normalized. The "center" and "scale" parameter tensors are created with the same shape as the axis that is normalized. The default of this parameter is '-1', i.e. the normalization, re-centering and re-scaling is over the last axis only.

I think it would be great to adapt the implementation to the behavior of the Tensorflow 1.x implementation in contrib.layers.layer_norm. There, 2 arguments "begin_norm_axis" and "begin_params_axis" where used. The former specifying the axis to normalize over, the latter to specify the axis to build the center and scale parameters (there called beta and gamma). The default values where "begin_norm_axis=1" (i.e. normalize over all axis but the first) and "begin_param_axis=-1", i.e. only re-scale and re-center along the last axis. 

**Will this change the current api? How?**

Only the implementation of keras.layers.LayerNormalization would be affected.  Note that keras.layers.LayerNormalization is not part of the "official" Keras API so it would not create any conflicts. 

The "axis" argument would be replaced by two arguments "norm_axis" and "param_axis".

**Who will benefit with this feature?**

This is especially relevant for convolutional networks. 
In my experience, the aforementioned default configuration works way better than normalizing, re-scaling and re-centering over a common set of axes.

Best wishes,
Philipp

**Comment by yhliang2018** [comment]
Keeping `axis=-1` arg only is the intended behavior. By default we should scale/center on the same axes as the axes that are being normalized, so this should default to the same as `axis`.  This also keeps the API of LayerNorm consistent with BatchNorm layer. 

TF addons also has [an implementation](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/normalizations.py) which can be taken as a reference.
